core/serializers.py

```diff
@@ -281,11 +285,27 @@ class Meta:
             'status'
         )
 
+    def isInstanceRunning(self, scheduled_event):
```

Comment:
It's python ... they do snake case for functions and variables.

Reply:
Updated